convert2rhel_insights_tasks/main.py

```diff
@@ -32,8 +32,10 @@
 C2R_LOG_FOLDER = "/var/log/convert2rhel"
 # Log file for convert2rhel
 C2R_LOG_FILE = "%s/convert2rhel.log" % C2R_LOG_FOLDER
-# Path to the convert2rhel report json file.
-C2R_REPORT_FILE = "%s/convert2rhel-pre-conversion.json" % C2R_LOG_FOLDER
+# Path to the convert2rhel pre conversion report json file.
+C2R_PRE_REPORT_FILE = "%s/convert2rhel-pre-conversion.json" % C2R_LOG_FOLDER
+# Path to the convert2rhel post conversion report json file.
+C2R_POST_REPORT_FILE = "%s/convert2rhel-post-conversion.json" % C2R_LOG_FOLDER
 # Path to the convert2rhel report textual file.
 C2R_REPORT_TXT_FILE = "%s/convert2rhel-pre-conversion.txt" % C2R_LOG_FOLDER
```

Comment:
shouldn't this also have `PRE` in a name, or is the txt file different from the pre-conversion analysis? What about post txt report?

Reply:
Added one for post-conversion and added the pre prefix. Thanks!